moviesurfer/data/download_data.py

```diff
@@ -0,0 +1,36 @@
+import logging
+import os
+import zipfile
+
+import wget
+
+
+# Function to download data
+def download_data(
+    output_dir="moviesurfer/data",
+    url="http://files.grouplens.org/datasets/movielens/ml-25m.zip",
+):
+    """
+    Downloads the MovieLens 25M Dataset, extracts it, and deletes the zip file.
+
+    :param output_dir: str, the output directory to save the downloaded data
+    :return: None
+    """
+    logging.basicConfig(level=logging.DEBUG)
+
+    # check if movies.csv file exists in output_dir
+    movies_csv_file = os.path.join(output_dir, "ml-25m/movies.csv")
+
+    if os.path.exists(movies_csv_file):
+        logging.info("Data already exists in the output directory.")
+        return
```

Comment:
Maybe warninng here `warning.warn`? Or even an exception. Otherwise the function will be completed just fine, but users won't know that nothing happened. Also maybe logging.warn

Reply:
The exception way felt like ok to me:)